modules/administration/pages/ssl-certs-imported.adoc

```diff
@@ -81,7 +81,7 @@ To replace the certificates, you can replace the installed CA certificate with t
 
 .Procedure: Replacing Existing Certificates
 
-. On the {productname} Server, at the command prompt, call the command `mgr-ssl-cert-setup` and provide the certificates as parameters:
+. On the {productname} container host, at the command prompt, call [command]``mgr-ssl-cert-setup`` and provide the certificates as parameters:
 +
 ----
 mgrctl exec -- mgr-ssl-cert-setup --root-ca-file=<Path_to_Root_CA_Certificate> --server-cert-file=<Server_Cert_File> --server-key-file=<Server_Key_File>
```

Comment:
It may happen that this command require a terminal as it might ask for a password.
I would suggest to add options `-ti` to "mgrctl exec"

```
mgrctl exec -ti -- mgr-ssl-cert-setup --root-ca-file=<Path_to_Root_CA_Certificate> --server-cert-file=<Server_Cert_File> --server-key-file=<Server_Key_File>
```

Reply:
using proposal via external commit.